### PR TITLE
Added tooltips for virtual product

### PR DESCRIPTION
**Pull request by thierrymarianne**
| Questions | Answers |
| --- | --- |
| Branch? | develop |
| Description? | Add tooltips to virtual product fields |
| Type? | improvement |
| Category? | BO |
| BC breaks? | no |
| Deprecations? | no |
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-1240 |
| How to test? | Create a new virtual product, switch to virtual product tab and check the tooltips |

### CODE REVIEW

File: src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig [hunk 1]
```diff
@@ -373,6 +373,8 @@
                         <div class="col-md-12">
                           <fieldset class="form-group">
                             <label class="form-control-label">{{ form.step3.virtual_product.file.vars.label }}</label>
+                            <span class="help-box" data-toggle="popover"
+                                  data-content="{{ "Upload a file from your computer (8.00 MB max.)"|trans({}, 'Admin.Catalog.Help') }}" ></span>
```
Comment (PrestaEdit) [inline src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig]: The post size is a fixed value ?

Comment (thierrymarianne) [inline src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig]: It's a max value.

Comment (PrestaEdit) [inline src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig]: I rewrite my ask, so: "The max value is a fixed value ?"

Comment (thierrymarianne) [inline src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig]: Max value is now retrieved from PHP config. Thanks.

File: src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig [hunk 2]
```diff
@@ -373,6 +373,8 @@
                         <div class="col-md-12">
                           <fieldset class="form-group">
                             <label class="form-control-label">{{ form.step3.virtual_product.file.vars.label }}</label>
+                            <span class="help-box" data-toggle="popover"
+                                  data-content="{{ "Upload a file from your computer %maxUploadSize% max.)"|trans({'%maxUploadSize%': max_upload_size}, 'Admin.Catalog.Help') }}" ></span>
```
Comment (AlexEven) [inline src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig]: We're missing a parenthesis here:
Upload a file from your computer **(**%maxUploadSize% max.)

Thank you!

Comment (thierrymarianne) [inline src/PrestaShopBundle/Resources/views/Admin/Product/form.html.twig]: The missing parenthesis has been added.  Thank you @AlexEven.

**Comment by vincentbz** [comment]
Tested and ok, thanks !

**Comment by maximebiloe** [comment]
Thanks

**Pull request merged**